paddle/fluid/framework/CMakeLists.txt

```diff
@@ -129,11 +129,13 @@ cc_test(version_test SRCS version_test.cc DEPS version)
 
 cc_library(proto_desc SRCS var_desc.cc op_desc.cc block_desc.cc program_desc.cc DEPS shape_inference op_info operator glog version)
 
-if(NOT WIN32)
-cc_library(ngraph_bridge SRCS ngraph_bridge.cc DEPS operator framework_proto ngraph)
-cc_library(ngraph_operator SRCS ngraph_operator.cc DEPS ngraph_bridge operator op_info device_context tensor scope glog
-  shape_inference data_transform lod_tensor profiler)
-endif(NOT WIN32)
+if(WITH_NGRAPH)
+  if(NOT WIN32)
```

Comment:
Since windows does not support `WITH_NGRAPH` now, 
https://github.com/PaddlePaddle/Paddle/blob/45fb357b60fac4ae0fe0c5e3c6d00bd5ea413a3b/cmake/external/ngraph.cmake#L17-L22
line133 and 137 could be removed.